src/catala/surface/lexer.ml

```diff
@@ -535,24 +536,24 @@ let lex_law (lexbuf : lexbuf) : token =
   let prev_lexeme = Utf8.lexeme lexbuf in
   let prev_pos = lexing_positions lexbuf in
   match%sedlex lexbuf with
-  | "/*" ->
+  | "```catala" ->
       is_code := true;
       code_string_acc := "";
       BEGIN_CODE
   | eof -> EOF
-  | "@@", Star white_space, "Master file", Star white_space, "@@" -> MASTER_FILE
-  | "@@", Star white_space, "Begin metadata", Star white_space, "@@" -> BEGIN_METADATA
-  | "@@", Star white_space, "End metadata", Star white_space, "@@" -> END_METADATA
-  | ( "@@",
+  | '#', Star white_space, "Master file" -> MASTER_FILE
+  | '>', Star white_space, "Begin metadata" -> BEGIN_METADATA
+  | '>', Star white_space, "End metadata" -> END_METADATA
+  | ( '>',
       Star white_space,
       "Include:",
       Star white_space,
-      Plus (Compl '@'),
+      Plus (Compl ('@' | '\n')),
       Star white_space,
       Opt ('@', Star white_space, "p.", Star white_space, Plus '0' .. '9', Star white_space),
-      "@@" ) ->
+      '\n' ) ->
       let extract_components =
-        R.regexp "@@\\s*Include\\:\\s*([^@]+)\\s*(@\\s*p\\.\\s*([0-9]+)|)@@"
+        R.regexp ">\\s*Include\\:\\s*([^@\\n]+)\\s*(@\\s*p\\.\\s*([0-9]+)|)"
```

Comment:
So why are there `@` left in this regexp ?

Reply:
It's because I left the possibility to add something of the form `> Include: filename @ p.1234`.

line 553:
```ocaml
Opt ('@', Star white_space, "p.", Star white_space, Plus '0' .. '9', Star white_space),
```

Reply:
Oh okay that's actually perfect :)